canonical-web-and-design/react-components
# Scope searchbox amends to search and filter

## Done

Scope searchbox amends to search and filter

## QA

### Storybook

To see rendered examples of all react-components, run:

```shell
yarn start
```

### QA

- View Storybook
- View SearchBox component and verify everything looks correct

Fixes #281
